examples/tensorflow/classification/datasets/preprocessing_selector.py

```diff
@@ -21,6 +21,7 @@
 PREPROCESSING_FN_MAP = {
     'imagenet2012': imagenet_preprocess_image,
     'cifar10': cifar10_preprocess_image,
+    'mock_cifar10': cifar10_preprocess_image,
```

Comment:
@kshpv, @andrey-churkin Could you provide the mock dataset for tests w/o changing the example code?

Reply:
Done